docs/tutorials/tutorial-0.rst

```diff
@@ -1,175 +1,65 @@
-Tutorial 0 - Sandbox
-====================
+Tutorial 0 - Getting Started
```

Comment:
Tutorials are the 'wordy' things. Having a step-by-step "here is how to setup pyenv, virtualenv, batavia" makes sense for a tutorial. 

I bumped the Sandbox Tutorial to 1, because the GettingStarted is now 0. Given we don't have a 'actually do something useful' tutorial as yet, i'm not sure how else the numbering could work.